src/Server.php

```diff
@@ -90,6 +90,14 @@ public function run(string $raw): ?string
         return $this->single($input);
     }
 
+    /**
+     * @return string[]|null
+     */
+    public function getMethods(): ?array
```

Comment:
I think the methods attribute cannot be null under any circumstances, better type to `array`